.setup/update_system.sh

```diff
@@ -23,9 +23,25 @@ if [ -d "${CURRENT_DIR}/../.vagrant" ]; then
     VAGRANT=1
 fi
 
+APT=true
+
+# check if apt upgrades have been disabled
+for flag in "$@"; do
```

Comment:
as opposed to being a runtime flag, I would suggest making it be a flag that gets set in a JSON file, as I think either a sysadmin would always want this or never want it, and not just sometimes not want it.

Reply:
The configuration is now in a JSON file with default behavior of not doing upgrades, if enabled, default config is not to override prompts, if it is set to true then the default behavior is to keep the old configuration file. I've kept the flag, maybe this can come handy if one wants to run `INSTALL_SUBMITTY.sh` such that it's guaranteed to be prompt-less at least from the side of `update_system.sh` irrespective of the configuration in `update_settings.json`. `CONFIGURE_SUBMITTY.py` silently creates `update_settings.json`, otherwise, it has to be manually created, this ensures that only those sys admins use this who really want it. If `update_settings.json` doesn't exist, upgrades don't happen.